server/routes/adminDonors.ts

```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
+import { getDonorsByCampaignId } from '../controllers';
+
+const adminDonors = Router();
+
+adminDonors.get('/campaign/donors/:id', getDonorsByCampaignId);
```

Comment:
Is it allowed for anyone to request this endpoint? 
And you need to fix the test depend on that